mysql-test/suite/galera_3nodes/t/galera_duplicate_primary_value.test

```diff
@@ -0,0 +1,84 @@
+--source include/galera_cluster.inc
+--source include/have_debug.inc
+--source include/have_debug_sync.inc
+--source include/big_test.inc
+--source include/force_restart.inc
+
+--let $galera_connection_name = node_3
+--let $galera_server_number = 3
+--source include/galera_connect.inc
+
+--connect node_1a, 127.0.0.1, root, , test, $NODE_MYPORT_1
+--connect node_1b, 127.0.0.1, root, , test, $NODE_MYPORT_1
+--connect node_1c, 127.0.0.1, root, , test, $NODE_MYPORT_1
+--connect node_1d, 127.0.0.1, root, , test, $NODE_MYPORT_1
+--connect node_1e, 127.0.0.1, root, , test, $NODE_MYPORT_1
+
+--connection node_1
+CREATE TABLE t1(a int not null primary key auto_increment, b int) engine=innodb;
+INSERT INTO t1(b) VALUES (1);
+
+--connection node_1c
+begin;
+insert into t1 values (2,2);
+
+--connection node_1d
+begin;
+insert into t1 values (3,3);
+
+--connection node_1a
+SET GLOBAL DEBUG_DBUG='+d,wsrep_after_kill';
+
+--connection node_2
+--send insert into t1 values (2,6);
+
```

Comment:
why --send here? this query will complete in node_2 without --send

Reply:
I will fix.